src/DynamoCore/Models/WorkspaceModel.cs

```diff
@@ -1284,7 +1284,7 @@ public ModelBase GetModelForElement(XmlElement modelData)
                 string.Format("Unhandled model type: {0}", helper.ReadString("type", modelData.Name)));
         }
 
-        internal ModelBase GetModelInternal(Guid modelGuid)
+        public ModelBase GetModelInternal(Guid modelGuid)
```

Comment:
Only after both of you make an agreement about that =P
[Ben's answer](https://github.com/DynamoDS/Dynamo/pull/3855#discussion_r25923747)